web/src/components/netflow-record/record-field.tsx

```diff
@@ -44,22 +44,26 @@ export const RecordField: React.FC<{
     );
   };
 
-  const kubeObjContent = (value: string | undefined, kind: string | undefined, ns: string | undefined) => {
-    if (value && ns && kind) {
+  const kubeObjContent = (value: string | undefined, kind: string | undefined, ns: string | undefined, ip: string) => {
+    if (value && kind) {
       return (
         <div className="force-truncate">
           <ResourceLink className={size} inline={true} kind={kind} name={value} namespace={ns} />
           <div className="record-field-tooltip">
-            <h4>Namespace</h4>
-            <span>{ns}</span>
-            &nbsp;
+            {ns && (
+              <>
+                <h4>Namespace</h4>
+                <span>{ns}</span>
+                &nbsp;
+              </>
+            )}
             <h4>{kind}</h4>
             <span>{value}</span>
           </div>
         </div>
       );
     } else {
-      return <div></div>;
+      return <div>{ip}</div>;
```

Comment:
That would be strange to have the ip address here since we also have the Source / Destination `Address` column

Reply:
yeah, I was hesitating on this one. I think it's interesting when you have the default columns (hence no IP) so it avoid blank cells. But when you add the IP columns, then it's weird.

Reply:
Actually, I'll remove it, it's actually weird because it makes it like the "Name" is an IP, but  if you filter with that IP on "Name" you would have no result, which sounds bogus. Also it can be surprising with sorts. I'll remove then. It's more a topic for the [super-columns](https://issues.redhat.com/browse/NETOBSERV-135).

Reply:
Now that the nodes ip are also resolved, are we still having many cases where an IP can not be resolved to an object?

Reply:
not as many as before, but I've seen some of them and to be honest I don't know what they are

Reply:
@jpinsonneau done, IP removed from there